hsm-gateway-applications/node/package.json

```
@@ -0,0 +1,36 @@
+{
+    "name": "gateway-hsm-sample",
+    "version": "0.0.1",
+    "description": "",
+    "main": "dist/hsm-sample.js",
+    "engines": {
+        "node": "^14.15.0 || ^16.13.0"
+    },
+    "scripts": {
+        "prepare": "npm run build",
+        "build": "npm-run-all clean compile lint",
+        "clean": "rimraf dist",
+        "compile": "tsc",
```

I would have the `npm install` just run the TypeScript compile, not the linting.
```suggestion
        "build": "tsc",
        "prepare": "npm run build",
        "clean": "rimraf dist",
```